Data/market-hours/market-hours-database.json

```diff
@@ -429,7 +429,7 @@
       "friday": [
         {
           "start": "00:00:00",
-          "end": "1.00:00:00",
+          "end": "16:00:00",
```

Comment:
Should end time be `Friday 5:45 p.m.` since `America/Chicago` is CT, same for cbot bellow? ref https://www.cmegroup.com/trading-hours.html

Reply:
The reason I set it to 16:00 is because I saw no assets closing past this time. All of the listed closing times matches with what's on the table, not the description of the closing hours -> table ref: CME Globex (Weekday)

Regarding the description, I have a feeling that that's for clearing/settlement -> reason I say that is because it says: "products submitted via CME Clearport Clearing". I can't find an asset with a trade halt greater than 16:00